pyproject.toml

```diff
@@ -67,6 +67,7 @@ module = "pyarr"
 disallow_untyped_defs = true
 
 [tool.isort]
+profile = "black"
 multi_line_output = 3
```

Comment:
The rest of this is no longer needed, all of these below are the defaults for the black profile I believe.

Reply:
Ah ok cool will remove that.